templates/chat.mustache

```diff
@@ -69,7 +83,8 @@
       })
       .then(({token}) => token)
   ]);
-
+  const completedActivities = JSON.parse('{{completedActivities}}');
+  const uncompletedActivities = JSON.parse('{{uncompletedActivities}}');
```

Comment:
```suggestion
  const completedActivities = JSON.parse(decodeHtml('{{completedActivities}}'));
  const uncompletedActivities = JSON.parse(decodeHtml('{{uncompletedActivities}}'));
```
It seems Mustache HTML encodes strings. Thus, we might need to decode them first with something like `decodeHthml`, for example:
```javascript
function decodeHtml(html) {
    var txt = document.createElement("textarea");
    txt.innerHTML = html;
    return txt.value;
}
```